sematic/ui/src/components/RunPanel.tsx

```diff
@@ -104,7 +105,18 @@ export default function RunPanel(props: {
             <Box sx={{ paddingBottom: 3, gridColumn: 1 }}>
               <Box marginBottom={3}>
                 <Typography variant="h6">{selectedRun.name}</Typography>
-                <CalculatorPath calculatorPath={"ID: " + selectedRun.id} />
+                <Typography fontSize="small" color="GrayText" component="span">
+                  <code style={{ fontSize: 12 }}>
+                  ID: {selectedRun.id}
+                  {
+                    selectedRun.original_run_id &&
+                    <>
+                      &nbsp;(cloned from&nbsp;
+                      <RunId runId={selectedRun.original_run_id} trim />)
+                    </>
+                  }
+                  </code>
+                </Typography>
```

Comment:
@chance-an Please advise on how to refactor this. Currently the theme of the link is lost, and it appears as clear text.

Reply:
The theme of the link will emerge if we implement it as a link :) 

I have done it in https://github.com/sematic-ai/sematic/pull/397.